docs/models/dki.rst

```diff
@@ -4,6 +4,35 @@
 Diffusion kurtosis imaging (DKI)
 ################################
 
+Water diffusion in the brain is affected by the physical structures that make up neurons and organelles.
+Instead of freely diffusing through space, water encounters barriers from myelin, cell membranes and other structures that introduce non-Gaussian features into the water diffusion distribution.
+The Diffusion Kurtosis Imaging (DKI; :cite:p:`jensen2005dki`) model extends the of the DTI model by adding an additional 15 parameters that capture the deviations from Gaussianity missed when fitting the simple 6 parameter DTI model.
```

Comment:
This is the name used in those papers:

```suggestion
The Diffusional Kurtosis Imaging (DKI; :cite:p:`jensen2005dki`) model extends the of the DTI model by adding an additional 15 parameters that capture the deviations from Gaussianity missed when fitting the simple 6 parameter DTI model.
```

I'd probably use that everywhere you spell out "DKI" (e.g., the title of this section)